public/manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Lets change these guys
```suggestion
  "short_name": "Horse Link",
  "name": "Horse Link",
```